code/game/sound.dm

```diff
@@ -22,6 +26,65 @@
 		0, // Flags (1 = Auto Direct, 2 = Auto Room, 4 = Auto RoomHF)
 	)
 	environment = SOUND_ENVIRONMENT_NONE //Default to none so sounds without overrides dont get reverb
+	var/atom/attached
+	var/list/listeners = list()
+	var/maxdist = 0
```

Comment:
Document all these.

Also, it looks like they don't clear properly if `attached` or whatever else gets deleted, meaning we're going to get hard dels.